ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -47,7 +47,7 @@ packet)
     # shellcheck source=/dev/null
     source "${SHARED_DIR}/ds-vars.conf"
     copy_test_binaries
-    export TEST_PROVIDER=\"\"
+    export TEST_PROVIDER='{"type":"baremetal"}'
```

Comment:
I had to escape the quotes when I was testing this the other day, as this is passed over ssh later on

https://github.com/openshift/release/pull/16986/files#diff-7b4946fc316154f2f0fe2beac2371cf82bc14d8071298890e8a014417e88159fR50

Reply:
Good catch, thanks!